kube/deployment.go

```diff
@@ -109,9 +109,16 @@ func replicaCheck(role *model.Role, controller *helm.Mapping, service helm.Node,
 		controller.Add("_oddReplicas", fail, helm.Block(block))
 	}
 
-	failLimitFactor := fmt.Sprintf(`{{ fail "The memory limit factor must be at least 1" }}`)
-	blockLimitFactor := fmt.Sprintf("if lt (int .Values.sizing.memory.limit_factor) 1")
-	controller.Add("_minLimitFactor", failLimitFactor, helm.Block(blockLimitFactor))
+	if settings.UseMemoryLimits {
+		failMemLimitFactor := fmt.Sprintf(`{{ fail "The memory limit factor must be at least 1" }}`)
+		blockMemLimitFactor := fmt.Sprintf("if lt (int .Values.sizing.memory.limit_factor) 1")
+		controller.Add("_minMemLimitFactor", failMemLimitFactor, helm.Block(blockMemLimitFactor))
+	}
+	if settings.UseCPULimits {
+		failCPULimitFactor := fmt.Sprintf(`{{ fail "The memory limit factor must be at least 1" }}`)
```

Comment:
The message here should be about CPU, not memory - correct?

Reply:
Definitely. Thanks for the catch. Will fix.